### ISSUE TITLE
Qv2ray proxy group saving issue

### ISSUE THREAD

**Issue by s-kk** [Qv2ray/Qv2ray#848]
- [x] **I'm sure I've tried to reproduce the problem several times and will provide the system environment, each software and its version that is involved in the problem.**

## Briefly describe this bug：
Proxy or proxy group settings won't be saved after clicking 'OK' in 'Group Editor' dialog.

## How to reproduce this bug：

1. change proxy/proxy group settings (e.g adding subscriptions or proxy servers)
2. apply changes and close server/group editing dialog
3. shutdown OS or terminate Qv2ray with ```kill```
4. restart Qv2ray, and proxy/proxy group settings were not saved.

### If it's a connection issue, has it happened in other V2Ray clients?
Not a connection issue, but it happens on both Linux and Windows client. MacOS client has not been tested. 

### Related screenshots or logs

N/A

```
Please paste your Qv2ray log here:


```

</details>

### Open Preferences -> About, and enter the following info

```
Version: 2.6.1:5779
Build info: Qv2ray built from Github Action
Extra build info: qt5.15.0-6e03550de328ed1f65b28756329ce8484bf66f88-ci.windows-latest
```

### Operating systems

- [x] Linux
- [ ] macOS
- [x] Windows
 
### Installation source
 
- [ ] ArchLinuxCN - Stable
- [ ] ArchLinuxCN - Dev-git
- [x] ArchLinux AUR - Stable
- [x] ArchLinux AUR - Development
- [ ] Fedora - Stable
- [ ] Fedora - Preview
- [ ] openSUSE - Stable
- [ ] openSUSE - Preview
- [ ] Flathub
- [ ] Snap
- [ ] Homebrew Cask
- [ ] Scoop / Chocolatey / Github Release
- [ ] Compile manually

### Installed plugins, versions and sources
N/A

## Additional Info

*Please hide your server address and UUID if you wish to post the vmess string or your connection setting.*

**Comment by kidonng** [comment]
配置是在**正常**退出程序时保存的，强行停止进程不会保存。
正常关机的情况下，Qv2ray 在 Windows 和 Linux 都能正确保存配置。可以尝试清空配置看看能否复现。

**Comment by s-kk** [comment]
> 
> 
> 配置是在**正常**退出程序时保存的，强行停止进程不会保存。
> 正常关机的情况下，Qv2ray 在 Windows 和 Linux 都能正确保存配置。可以尝试清空配置后看看能否复现。
> 此外系统/DE设置中的“保存对话”或是“注销时保存应用”可能会造成影响，也请确认是否关闭了它们。

Just tested with Windows, the issue persists with a normal OS shutdown. Settings won't be saved. It's advised to change the policy of saving and updating configurations.

**Comment by DuckSoft** [comment]
Uh, it's a problem. We should sync our nodes when the configuration is "saved".
Let's do some patch work. @U-v-U

**Comment by U-v-U** [comment]
Should have been fixed in the v2.6.3.
